### ISSUE TITLE
Rethink Free France rebellion focuses

### ISSUE THREAD

**Issue by luceSPT** [SPT-Mod-Dev/SPT-Mod-Dev#104]
1) Decisions unlocked by "intervention in X" don't appear to be fireable only once
2) Maybe the whole tree is kinda dumb when Vichy doesn't exist.

**Comment by chaffordh** [comment]
Consider adding a state modifier with increased resistance effects

**Comment by chaffordh** [comment]
added civ/mil/manpower decisions, consider future work
